sandec/get_dll.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+if ! [ -x "$(command -v cabextract)" ]; then
+    echo "cabextract not found."
+    exit 1;
+fi
+
+if ! [ -x "$(command -v unshield)" ]; then
+    echo "unshield not found."
+    exit 1;
+fi
+
+mkdir work_tmp;
+cd work_tmp;
```

Comment:
Here's a little bash trick for you.

```bash
TMP=$(mktemp -d)
trap "rm -rf $TMP" EXIT
```
You'll have a temporary directory that won't collide and will automatically get cleaned up.

Reply:
Cool :)